packages/qwik-city/buildtime/vite/dev-server.ts

```diff
@@ -128,6 +128,11 @@ export function ssrDevMiddleware(ctx: BuildContext, server: ViteDevServer) {
               res.setHeader(key, value);
             });
 
+            const cookieHeaders = requestEv.cookie.headers();
```

Comment:
guess this changes shouldn't be here, right? 👼  looks like https://github.com/BuilderIO/qwik/pull/3276/files